docs-2.0/20.appendix/error-code.md

```diff
@@ -0,0 +1,181 @@
+|`-2080`| Invalid variable|
+|`-2081`| Variable value and type do not match.|
+|`-3001`| Consensus cannot be reached during an election.|
+|`-3002`| Key already exists.|
+|`-3003`| Data type mismatch|
+|`-3004`| Invalid field value|
+|`-3005`| Invalid operation|
+|`-3006`| Current value is not allowed to be empty.|
+|`-3007`| Field value must be set if the field value is `NOT NULL` or has no default value.|
+|`-3008`| The value is out of the range of the current type.|
+|`-3010`| Data conflict|
+|`-3011`| Writes are delayed|
+|`-3021`| Incorrect data type|
+|`-3022`| Invalid VID length|
+|`-3031`| Invalid filter|
+|`-3032`| Invalid field update|
+|`-3033`| Invalid KV storage|
+|`-3034`| Peer invalid|
+|`-3035`| Out of retries|
+|`-3036`| Leader change failed|
+|`-3037`| Invalid stat type|
+|`-3038`| VID is invalid.|
+|`-3040`| Failed to load meta information.|
+|`-3041`| Failed to generate checkpoint.|
+|`-3042`| Generating checkpoint is blocked.|
+|`-3043`| Data is filtered.|
+|`-3044`| Invalid data|
+|`-3045`| Concurrent write conflicts on the same edge|
+|`-3046`| Concurrent write conflict on the same vertex |
+|`-3047`| Lock is invalid.|
+|`-3051`| Invalid task parameter.|
+|`-3052`| The user canceled the task.|
+|`-3053`| Task execution failed|
+|`-3060`| Execution plan was cleared.|
+|`-3070`| The heartbeat process was not completed when the request was received.|
+|`-3071`| Out-of-date heartbeat received from the old leader (the new leader has been elected).|
+|`-3073`| Concurrent write conflicts with later requests|
+|`-3061`| Client and server versions are not compatible|
+|`-3062`| Failed to get ID serial number|
+|`-3500`| Unknown partition|
+|`-3501`| Raft logs lag behind.|
+|`-3502`| Raft logs are out of date.|
+|`-3503`| Heartbeat messages are out of date.|
+|`-3504`| Unknown additional logs|
+|`-3511`| Waiting for the snapshot to complete|
+|`-3512`| There was an error sending the snapshot.|
+|`-3513`| Invalid receiver|
+|`-3514`| Raft did not start.|
+|`-3515`| Raft has stopped.|
+|`-3516`| Wrong role|
+|`-3521`| Write to a WAL failed|
+|`-3522`| The host has stopped.|
+|`-3523`| Too many requests|
+|`-3524`| Persistent snapshot failed|
+|`-3525`| RPC exception|
+|`-3526`| No WAL logs found|
+|`-3527`| Host suspended|
+|`-3528`| Writes are blocked.|
+|`-3529`| Cache overflow|
+|`-3530`| Atomic operation failed|
+|`-3531`| Leader lease expired|
+|`-3532`| Data has been synchronized on Raft.|
+|`-4001`| Drainer logs lag behind.|
+|`-4002`| Drainer logs are out of date.|
+|`-4003`| The drainer data storage is invalid.|
+|`-4004`| Graph space mismatch|
+|`-4005`| Partition mismatch|
+|`-4006`| Data conflict|
+|`-4007`| Request conflict|
+|`-4008`| Illegal data|
+|`-5001`| Cache configuration error|
+|`-5002`| Insufficient space|
+|`-5003`| No cache hit|
+|`-5005`| Write cache failed|
+|`-7001`| Number of machines exceeded limit|
+|`-7002`| Failed to resolve certificate|
+|`-8000`| Unknown error|
```

Comment:
Some have periods and some don't, is this expected?

Reply:
Periods only for complete sentences. Rechecked them all, thanks babe.